docs/configuring-ingest.asciidoc

```diff
@@ -40,6 +40,17 @@ and then viewing `ingest/pipeline/definition.json`.
 
 To disable this, or any other pipeline, set `output.elasticsearch.pipeline: _none`.
 
+`apm_error_grouping_name`::
```

Comment:
L38-41 snuck in the middle of the table. Can you move L38-41 below the new content that you've added.

Reply:
Oops - fixed, thanks.